package/MDAnalysis/coordinates/XDR.py

```diff
@@ -195,18 +197,18 @@ def _load_offsets(self):
 
         #  check if the location of the lock is writable.
         try:
-            with fasteners.InterProcessLock(lock_name) as filelock:
+            with FileLock(lock_name) as filelock:
                 pass
         except OSError as e:
             if isinstance(e, PermissionError) or e.errno == errno.EROFS:
                 warnings.warn(f"Cannot write lock/offset file in same location as "
                               f"{self.filename}. Using slow offset calculation.")
-                self._read_offsets(store=True)
+                self._read_offsets(store=False)
```

Comment:
good!

(I think this also explain why you changed the check for a warning in the tests)